spec.html

```diff
@@ -20271,9 +20285,9 @@ <h1>Runtime Semantics: PropertyDefinitionEvaluation</h1>
         1. Let _propKey_ be the result of evaluating |PropertyName|.
         1. ReturnIfAbrupt(_propKey_).
         1. Let _scope_ be the LexicalEnvironment of the running execution context.
-        1. Let _closure_ be ! AsyncFunctionCreate(~Method~, |UniqueFormalParameters|, |AsyncFunctionBody|, _scope_).
+        1  Let _name_ be ! ToFunctionName(_propKey_).
```

Comment:
```suggestion
        1. Let _name_ be ! ToFunctionName(_propKey_).
```
(Change space to dot.)